frontend/public/lib/imageOperations.ts

```diff
@@ -131,13 +131,32 @@ const drawImageOnCanvas = (image, canvas) => {
   }
 };
 
-export async function blobFromObjectUrl(objectUrl) {
-  var a = new FileReader();
-  const blob = await fetch(objectUrl).then((r) => r.blob());
-  return new Promise(function (resolve) {
-    a.onload = function (e) {
-      resolve(e.target!.result);
+export async function blobFromObjectUrl(objectUrl: string): Promise<string> {
+  try {
+    const response = await fetch(objectUrl);
+    const originalBlob = await response.blob();
+
+    const options = {
+      maxSizeMB: 0.5,
```

Comment:
Previously this was a parameter that was used with 1 as value in two places and 0.5 in 3 places. Although 0.5 seems good to me as we need to keep it small.